src/lib/core/ripple/ripple-renderer.ts

```diff
@@ -246,10 +246,13 @@ export class RippleRenderer {
 
   /** Function being called whenever the trigger is being pressed using mouse. */
   private onMousedown = (event: MouseEvent) => {
+    // Screen readers will fire fake mouse events for space/enter. Skip launching a
+    // ripple in this case for consistency with the non-screen-reader experience.
+    const isFakeMousedown = event.buttons === 0;
```

Comment:
Can we please use `isFakeMousedownFromScreenReader` from `@angular/cdk/a11y`?

Reply:
I wanted to avoid pulling in cdk/a11y since the idea is to pull the ripples out into their own module.

Reply:
I see. I don't really think that this means that we should hesitate using the CDK. Even if the ripples are moved into its own module, they can still have dependencies on the CDK. 

The goal is not make ripples independent from anything else. It should be just able for developers to explicitly use `@angular/material/ripple` without having all the other stuff.

e.g. see `@angular/material/core` does have deps on the CDK, and also the ripples already use `cdk/platform`.

Reply:
What I meant is that it seemed like overkill to bring in another CDK package for a one-liner method. I've switched it now.

Reply:
Yeah, totally see what you mean. My reasoning is just that the CDK is being pulled by NPM anyway and with tree shaking (which is quite common used) it doesn't really matter how big the `a11y` package is because only the one function will be pulled in.

Reply:
Not sure if I can make that decision though. Let's wait what Jeremy says on that. I'm a huge fan of consistency, and I'm worried that we'll duplicate logic which can be easily forgotten.